src/utils/SpokeUtils.ts

```diff
@@ -240,6 +240,23 @@ export function getV3RelayHash(relayData: V3RelayData, destinationChainId: numbe
   );
 }
 
+export function getV3RelayHashFromEvent(e: V3Deposit | V3Fill | SlowFillRequest): string {
```

Comment:
I'm pretty sure it's not necessary to define `relayData` here. Each of the types `V3Deposit | V3Fill | SlowFillRequest` are a superset of a `RelayData` type, and the hash function is abi-aware so it's smart enough to pick out the right fields and place them in the correct order - even if additional fields are supplied. So I think the entire function can probably just be removed, and the caller can call `getV3RelayHash()` directly.

Reply:
i think this is useful so you don't have to pass in the destination chain